### PR TITLE
feat/coworkers-invite-received

### PR DESCRIPTION
**Pull request by NeoxAzrot**
feat/coworkers-invite-received

### CODE REVIEW

File: app/settings/components/form-footer/form-footer.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export function FormFooter({ isPending, hasPreviewButton, isAbsolute = true }: T
       alignItems="center"
       justifyContent="center"
       className={cn("border-t border-greyscale-50/8", {
-        "absolute bottom-0 left-0 right-0 z-50 bg-spaceBlue-900 px-4 shadow-medium xl:px-8": isAbsolute,
+        "absolute bottom-0 left-0 right-0 z-50 bg-spaceBlue-900 px-4 py-4 shadow-medium xl:px-8": isAbsolute,
```
Comment (haydencleary) [inline app/settings/components/form-footer/form-footer.tsx:44]:
```suggestion
        "absolute bottom-0 left-0 right-0 z-50 bg-spaceBlue-900 p-4 shadow-medium xl:px-8": isAbsolute,
```

File: components/features/stacks/billing-create-stack/components/checkbox-item/checkbox-item.tsx [hunk 2]
```diff
@@ -48,14 +62,29 @@ export function CheckboxItem({
...
+              </Tooltip>
+            ) : (
+              <Checkbox
+                radius="full"
+                isSelected={selected}
+                className="pointer-events-none"
+                classNames={{
+                  wrapper: "after:bg-spacePurple-500",
+                }}
+              />
```
Comment (haydencleary) [inline components/features/stacks/billing-create-stack/components/checkbox-item/checkbox-item.tsx:86]: Maybe `Checkbox` could be extracted into a variable ?

File: translations/v2/en/pages/settings-billing/settings-billing-informations.json [hunk 3]
```diff
@@ -57,5 +57,17 @@
       "no": "No",
       "yes": "Yes"
     }
+  },
+  "invitation": {
+    "title": "You’ve been invited to the __name__ payout profile.",
+    "description": "Please accept only if you are a member of this company.",
```
Comment (haydencleary) [inline translations/v2/en/pages/settings-billing/settings-billing-informations.json:63]:
```suggestion
    "description": "Please only accept if you are a member of this company.",
```

File: translations/v2/en/pages/settings-billing/settings-billing-informations.json [hunk 4]
```diff
@@ -57,5 +57,17 @@
...
+    "buttons": {
+      "accept": "Accept",
+      "decline": "Decline"
+    },
+    "alert": {
+      "success": "You successfully joined __name__",
```
Comment (haydencleary) [inline translations/v2/en/pages/settings-billing/settings-billing-informations.json:69]:
```suggestion
      "success": "You joined __name__ successfully",
```

**Review by haydencleary — CHANGES_REQUESTED** [review]

File: src/api/me/mutations.ts [hunk 5]
```diff
@@ -120,6 +120,21 @@ const useUpdatePayoutPreferences = ({ options = {} }: UseMutationProps<unknown,
   });
 };
 
+const useAcceptOrDeclineInvitation = ({
+  params,
+  options,
+}: UseMutationProps<unknown, { billingProfileId: string }, { accepted: boolean }>) => {
+  return useBaseMutation<{ accepted: boolean }, unknown>({
+    resourcePath: ME_PATH.BILLING_PROFILES_INVITATIONS(params?.billingProfileId || ""),
```
Comment (pixelfact) [inline src/api/me/mutations.ts:128]:
```suggestion
    resourcePath: ME_PATH.BILLING_PROFILES_INVITATIONS(params?.billingProfileId || ""),
    enabled: !!params?.billingProfileId,
```

**Review by pixelfact — APPROVED** [review]

**Pull request merged**